python/mleap/sklearn/logistic.py

```diff
@@ -88,7 +94,9 @@ def deserialize_from_bundle(self, transformer, node_path, node_name):
 
         attributes_map = {
             'coefficients': 'coef_',
-            'intercept': 'intercept_'
+            'coefficient_matrix': 'coef_',
+            'intercept': 'intercept_',
+            'intercept_vector': 'intercept_',
```

Comment:
probably we should have the if statement for the number of classes here as well, what do you think?

Reply:
You need to deserialize the transformer to know its number of classes. And you can't deserialize without already having this attribute map.